src/main/java/org/dcsa/bkg/model/transferobjects/BookingTO.java

```diff
@@ -17,7 +17,7 @@
   private LocationTO placeOfIssue;
 
   @Valid
-  @NotEmpty(message = "Commodities are required.")
+  @NotEmpty(message = "The property commodities is required.")
```

Comment:
Shouldn't it be `attribute` here instead of `property`? to be consistent.

Reply:
you're probably right. Changed it to `attribute`.